### PR TITLE
add autocomplete-property to Scale-TextField

### PR DESCRIPTION
**Pull request by niclas18**
close my issue #1358

### CODE REVIEW

File: packages/components/src/components/text-field/text-field.tsx [hunk 1]
```diff
@@ -105,6 +105,8 @@ export class TextField {
   @Prop() list?: string;
   /** (optional) the input should automatically get focus when the page loads. */
   @Prop() inputAutofocus?: boolean;
+  /** (optional) custom value for automplete HTML attribute */
+  @Prop() autocomplete?: string;
```
Comment (nowseemee) [inline packages/components/src/components/text-field/text-field.tsx:109]: can you rename it to `inputAutocomplete`, please? So it won't clash with the native attribute name.

File: packages/components/src/components/text-field/text-field.tsx [hunk 2]
```diff
@@ -105,6 +105,8 @@ export class TextField {
   @Prop() list?: string;
   /** (optional) the input should automatically get focus when the page loads. */
   @Prop() inputAutofocus?: boolean;
+  /** (optional) custom value for automplete HTML attribute */
```
Comment (nowseemee) [inline packages/components/src/components/text-field/text-field.tsx:108]:
```suggestion
  /** (optional) custom value for autocomplete HTML attribute */
```

File: packages/components/src/components/text-field/text-field.tsx (hunk 1 above)
Comment (niclas18) [reply packages/components/src/components/text-field/text-field.tsx]: Thanks for your feedback, I changed/add it in my last [commit](https://github.com/telekom/scale/pull/1400/commits/ea1b4a868bc70ad7ae6b0bd0863643a658967ccc).

File: packages/components/src/components/text-field/text-field.tsx (hunk 2 above)
Comment (niclas18) [reply packages/components/src/components/text-field/text-field.tsx]: Thanks for your feedback, I changed/add it in my last [commit](https://github.com/telekom/scale/pull/1400/commits/ea1b4a868bc70ad7ae6b0bd0863643a658967ccc).

**Comment by nowseemee** [comment]
Many thanks for your contribution, @niclas18! We'll release it early next week.

**Pull request merged**